### PR TITLE
Removes common-common ☠️ 🥀 

### PR DESCRIPTION
**Pull request by Rotorsoft**
This PR ends the common-common refactoring tasks 

## Link to Issue
Closes: #6205

## Description of Changes
- Relocates deamons in common-common to commonwealth/test/unit
- Relocates hardhat.config.ts - @ianrowan needs to review this
- Removes all refs to common-common
- Adds rootDir to service tsconfigs
- Adjusts scripts to new build structure
 
## Testing
- Tested on frick and discobot-staging envs

### CODE REVIEW

**Review by kurtisassad — APPROVED** [review]
LGTM 🚀

File: packages/commonwealth/webpack/webpack.prod.config.js [hunk 1]
```diff
@@ -35,7 +35,6 @@ module.exports = merge(common, {
         include: [
           path.resolve(__dirname, '../client'),
           path.resolve(__dirname, '../shared'),
-          path.resolve(__dirname, '../../common-common'),
           path.resolve(__dirname, '../../chain-events'),
```
Comment (timolegros) [inline packages/commonwealth/webpack/webpack.prod.config.js:38]: We can remove `../../chain-events` as well

**Review by timolegros — APPROVED** [review]

**Comment by CowMuon** [comment]
I see we have approvals from Timothee and Kurtis. @ianrowan did you have a chance to review hardhat location/possible slippage there?

**Comment by Rotorsoft** [comment]
We did it offline @CowMuon , he also suggested the last commit. Already deployed to frick/discobot-staging and testing PR #6296

**Pull request merged**